tests/gold_tests/continuations/double.test.py

```diff
@@ -50,11 +49,11 @@
     'map http://127.0.0.1:{0} http://127.0.0.1:{1}'.format(ts.Variables.port, server.Variables.Port)
 )
 
-numberOfRequests = randint(1000, 1500)
+numberOfRequests = 10
 
-# Make a *ton* of calls to the proxy!
+# Make calls to the proxy!
 tr = Test.AddTestRun()
-tr.Processes.Default.Command = 'ab -n {0} -c 10 http://127.0.0.1:{1}/;sleep 5'.format(numberOfRequests, ts.Variables.port)
+tr.Processes.Default.Command = 'for ((i=0; i<{0}; i++)); do curl -k https://127.0.0.1:{1}; done; sleep 5'.format(numberOfRequests, ts.Variables.port)
```

Comment:
* Do we still need the sleep here?
* Does it matter that the client requests will be made serially instead of in parallel?

Reply:
The sleep is necessary because traffic_ctl needs to process and output the metrics. Without it, TSStatIntIncrement may not register. 
Requests should be made in parallel. Good point. Changed